main.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+let path = require('path');
+
+// Import the Math class.
+let MyMath = require(path.join(__dirname, 'myMath'));
+
+let intVal = MyMath.toInt("10");
+console.log(`Integer val is ${intVal}`);
```

Comment:
kannste so machen für string concat, aber console.log/warn/error akzeptiert `n` argumente,
von daher tuts auch `console.log('Integer val is ', intVal);`